include/circt/Dialect/OM/OMOps.td

```diff
@@ -16,6 +16,8 @@
 include "circt/Dialect/OM/OMOpInterfaces.td"
 include "mlir/Interfaces/SideEffectInterfaces.td"
 include "mlir/IR/BuiltinAttributeInterfaces.td"
+include "mlir/Interfaces/InferTypeOpInterface.td"
```

Comment:
Nit: Header sorting.  Same above in `OMOps.h`, but at least group headers under same directory together please.